test/main/services/team/TeamProvisioningLaunchFailurePolicy.test.ts

```
@@ -0,0 +1,113 @@
+  it('keeps auto-clear policy narrow but accepts known recoverable suffixes', () => {
+    expect(
+      isAutoClearableLaunchFailureReason('Teammate was never spawned during launch.')
+    ).toBe(true);
+    expect(isAutoClearableLaunchFailureReason('process table is unavailable')).toBe(true);
+    expect(
+      isAutoClearableLaunchFailureReason(
+        'Teammate did not join within the launch grace window.; process table unavailable'
+      )
+    ).toBe(true);
+    expect(isAutoClearableLaunchFailureReason('model not found')).toBe(false);
+    expect(isAutoClearableLaunchFailureReason(undefined)).toBe(false);
```

ESLint

[error] Remove this redundant "undefined".

(sonarjs/no-undefined-argument)